src/plugins/cosmos/components/AmountToStake/AmountToStake.tsx

```diff
@@ -14,6 +15,7 @@ type AmountToStakeProps = {
   }
 }
 export const AmountToStake = ({
```

Comment:
Obviously this now is handling both stake and unstake copies.
Happy to implement renaming suggestions.